src/login.rs

```diff
@@ -1006,3 +1155,17 @@ fn reset_error_label(error_label: &gtk::Label) {
     error_label.set_text("");
     error_label.set_visible(false);
 }
+
+fn stringify_auth_code_type(code_type: AuthenticationCodeType) -> String {
+    match code_type {
+        AuthenticationCodeType::TelegramMessage(_) => gettext("Telegram"),
+        AuthenticationCodeType::Sms(_) => gettext("SMS"),
+        AuthenticationCodeType::Call(_) => gettext("Call"),
+        AuthenticationCodeType::FlashCall(_) => gettext("Flash Call"),
+        AuthenticationCodeType::MissedCall(_) => gettext("Missed Call"),
+    }
+}
+
+fn code_next_type_wait_text(seconds: i32) -> String {
```

Comment:
If it's just one line I don't think it's really worth having this function at all, gettext already aggregates equals strings automatically.